edx_proctoring/views.py

```diff
@@ -700,6 +700,8 @@ def get(self, request, course_id):
         """
         HTTP GET handler.
         """
+        # pylint: disable=too-many-statements
```

Comment:
I expect this to be temporary. I'll remove the additional logging statements once the performance work is done.